api/grpcserver/activation_service.go

```diff
@@ -42,12 +45,28 @@ func (s *activationService) Get(ctx context.Context, request *pb.GetRequest) (*p
 	atxId := types.ATXID(types.BytesToHash(request.Id))
 	atx, err := s.atxProvider.GetFullAtx(atxId)
 	if err != nil || atx == nil {
-		s.logger.With().Debug("failed to get the ATX", log.Err(err), log.Stringer("id", atxId))
+		s.logger.With().Error("failed to get ATX",
```

Comment:
I'm not sure we should "ERROR" here. I can request any ATX via GRPC.

The local node could not yet be fully synced - then this warrants no error I think. The remote node could request an invalid ID, which also isn't an error on the local nodes side, etc.

Reply:
you are right. i was possessed when i changed that.